langchain4j-local-ai/src/test/java/dev/langchain4j/model/localai/AbstractLocalAiInfrastructure.java

```diff
@@ -0,0 +1,100 @@
+package dev.langchain4j.model.localai;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.model.Image;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class AbstractLocalAiInfrastructure {
+
+    private static final String LOCAL_AI_IMAGE = "quay.io/go-skynet/local-ai:v1.40.0";
+
+    private static final String LOCAL_IMAGE_NAME = "tc-local-ai";
+
+    private static final String LOCAL_LOCAL_AI_IMAGE = String.format("%s:%s", LOCAL_IMAGE_NAME, DockerImageName.parse(LOCAL_AI_IMAGE).getVersionPart());
+
+    private static final List<String[]> CMDS = Arrays.asList(
+            new String[] {"curl", "-o", "/build/models/ggml-gpt4all-j", "https://gpt4all.io/models/ggml-gpt4all-j.bin"},
```

Comment:
The other test using this model fail because the answer is not the expected one.